src/platform/backends/qemu/firewall_config.cpp

```diff
@@ -94,6 +96,12 @@ const QString RETURN{QStringLiteral("RETURN")};
 const QString reject_with{QStringLiteral("--reject-with")};
 const QString icmp_port_unreachable{QStringLiteral("icmp-port-unreachable")};
 
+class FirewallException : public std::runtime_error
+{
+public:
+    using std::runtime_error::runtime_error;
```

Comment:
I mean something like 

```c++
FirewallException(const std::string& issue, const std::string& table, const std::string& failure, const std::string& output)
  : runtime_error{fmt::format("{}; Table: {}; Failure: {}; Output: {}", issue, table, failure, output)}
{}
```

Separating things with semi-colons again avoids translation issues.

Reply:
Right, makes sense.

Reply:
Fixed as you suggested.